docs/src/administration/sso.md

```diff
@@ -46,4 +46,4 @@ To resolve that, you need to:
 
 {{< premium-features/tiered "Elite" >}}
 
-Enforce your users to authenticate with your OpenID Connect provider. Please open a support ticket to enable SSO with your OpenID Connect provider.
+Enforce your users to authenticate with your OpenID Connect provider, such as Okta, Azure Active Directory or Ping Identity for example. Please open a support ticket to enable SSO with your OpenID Connect provider.
```

Comment:
Nitpick: if you use _such as_, you don't need _for example_ at the end of the sentence. The rest is just me taking this opportunity to better apply [our style guide](https://github.com/platformsh/platformsh-docs/blob/main/CONTRIBUTING.md).

```suggestion
Enforce your users to authenticate with your OpenID Connect provider,
such as Okta, Azure Active Directory or Ping Identity. 
To enable SSO with your OpenID Connect provider,
[contact Support](https://console.platform.sh/-/users/~/tickets). 
```